forms-flow-api-utils/src/formsflow_api/app.py

```diff
@@ -0,0 +1,62 @@
+"""App initialization.
+
+Initialize app and the dependencies.
+"""
+
+from formsflow_api.exceptions import BusinessException
+from formsflow_api.schemas import FormioRoleSchema
+from formsflow_api.services.external import FormioService
+from formsflow_api.utils import (
+    cache
+)
+from formsflow_api.utils.enums import FormioRoles
+
+def setup_jwt_manager(app, jwt_manager):
+    """Use flask app to configure the JWTManager to work for a particular Realm."""
+
+    def get_roles(a_dict):
+        resource = a_dict["resource_access"].get(app.config["JWT_OIDC_AUDIENCE"])
+        return resource["roles"] if resource else a_dict["roles"]
+
+    app.config["JWT_ROLE_CALLBACK"] = get_roles
+    jwt_manager.init_app(app)
+
+def collect_role_ids(app):
```

Comment:
I think we can move this def to somewhere else as we only keep defs needed for app start up in app.py